src/utils.ts

```diff
@@ -26,6 +26,18 @@ import { TDMLConfig } from './adapter/activateDaffodilDebug'
 const defaultConf = vscode.workspace.getConfiguration()
 let currentConfig: vscode.DebugConfiguration
 
+// if active termainl is one of these, use the active terminal instead of making a new one
+const allowedShells = [
```

Comment:
Only put in what you are going to create tests for.  If we only have tests for cmd and bash, then that's the only thing we should allow.

Reply:
gotcha will fix that now

Reply:
Do you think there should possibly be a `TODO` comment to add support for others?

Reply:
Well, what value add does it provide?

That people can type commands at the terminal using their preferred shell? 

If this is an expected thing under VSCode, i.e, that you can open a terminal in your preferred shell, then maybe we shouldn't be using the regular terminal window for our needs.

Why wouldn't we give them a separate terminal for their command line interactions, and the one we're controlling can be exactly what we want it to be and that we test. (Maybe it can be named Daffodil shell instead of "Terminal" even?) I don't know what's possible, but to me if we're issuing commands that have to work, and they require a terminal window to capture the output, then we need to constrain what shell is used, and that is in tension with a terminal window the user interacts with.

Reply:
Ah I see what you are seeing I will look into adding this. I believe it normally defaults to the shell or command being ran but will see if I could set to `Daffodil` or `daffodil-debugger`.

Reply:
Will this be broken if a VSCode user sets the default shell to something we're not expecting?
If so, then can we stipulate exactly which shell is used in a terminal we create?